tools/dds/dds-config/readme.md

```diff
@@ -0,0 +1,61 @@
+# dds-config tool
+
+## Goal
+This tool is used to configer DDS based device and ensure it operates correctly within a DDS environment.  
```

Comment:
`configure`

Reply:
"a DDS device" or "devices"

Reply:
As discussed better to use "Ethernet based devices" not "DDS". General comment throughout the file

Reply:
done